src/layout/verticalPanel/statusArea.ts

```diff
@@ -197,7 +197,7 @@ export class MsStatusArea extends Clutter.Actor {
         // All icons actors should have a single child.
         const mainChild = actor.get_children()[0] as Clutter.Actor | undefined;
         if (mainChild !== undefined) {
-            if (mainChild instanceof panel.AggregateMenu) {
+            if (mainChild instanceof (compareVersions(gnomeVersionNumber, parseVersion('43.0')) >= 0 ? panel.QuickSettings : panel.AggregateMenu)) {
```

Comment:
We want to check for gnome 43 and greater not only 43 you can use `gnomeVersionGreaterOrEqualTo`

Reply:
It does check for equal or greater to.

Reply:
Also, if I read the code correctly, `gnomeVersionGreaterOrEqualTo` requires using an instance of a `TypeWithGnomeVersion`, which I didn’t define. Maybe it is possible to do that, but I don’t have the knowledge of TypeScript to determine that or, if so, how it’s done.